hw/tests/config_streamer.py

```diff
@@ -8,15 +9,15 @@
 
 # Add the amplipi directory to PATH
 sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '../..')))
-from amplipi.eeprom import BoardInfo, BoardType, EEPROM, UnitType
 
-BOARD_REV_RE = re.compile(r"(\d+)([A-z])") # accept lower or uppser case change designator
-                                           # we will uppercase later
+BOARD_REV_RE = re.compile(r"(\d+)([A-z])")  # accept lower or upper case change designator
+# we will uppercase later
```

Comment:
Out of curiosity, what do you feel this is obscuring? I suggested this change in the first place so I'd like to know for my own future reference. I felt that since we were already making changes in this area of the file, since the auto-formatter added a space on this line already, and since in the commit you can clearly see it's just a change in the comment, that it was worth fixing the typo now.